src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -28,13 +28,6 @@
      [rn/view
       {:style (style/dot-divider theme)}])])
 
-(def ^:private default-token-symbols
-  {:ethereum "ETH"
-   :optimism "OP"
-   :arbitrum "ARB"
-   :base     "ETH"
-   :status   "ETH"})
-
```

Comment:
all the `quo/summary-info` changes are done to remove the dependency on this def.